src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -506,7 +508,7 @@ async Task OnStreamCanceled(InvocationRequest irq)
                 irq.Dispose();
             }
 
-            var readers = PackageStreamingParams(ref args, out var streamIds);
```

Comment:
Why did you turn the readers local into a _readers field? It only appears to be used in PackageStreamingParams and LaunchStreams, and LaunchStreams always seems to be called directly after PackageStreamingParams, so I don't see the need.

Reply:
This was from an intermediate state of the pr that I missed during clean up.